### ISSUE TITLE
[BUG] Every first use after loggedin results in a blank screen

### ISSUE THREAD

**Issue by zazy** [G-dH/vertical-workspaces#79]
**Describe the bug**
Everytime I login to my local account and I try to use V-Shell for the first time by pressing the "Super" key on my keyboard, the result is the following: the bottom docking bar have a strange "ghost" (like several dockbars overlaps at a small interval), the currently running applications are not shown. The search doesn't works.
Hitting the "Super" key for a second time will hide everything and return to the desktop. Hitting the "Super" key again will show V-Shell normally

**Basic debug information**
 - Did disabling all other extensions help? **No**
 - Did you find conflicting extension? **No**

**To Reproduce**
1. Login
2. Press the "Super" key

**Expected behavior**
I expect V-Shell show correctly on the firse "Super" key press

**Screenshots**
I attach a video. When in the video you see the black screen, there is the moment the strange behaviour is. Don't know why video recorded a black screen. In the video I've run four terminals with Ctrl+T key to avoid launching them from the overview.
[Video del 18-05-2023 10:18:57.webm](https://github.com/G-dH/vertical-workspaces/assets/10668286/3d2a1643-94c0-4694-9899-ef8ab6ab08f4)

**System:**
 - OS: Ubuntu 22.04.2
 - Version of V-Shell extension: 29
 - Source of installation: extension.gnome.org
 - Other installed and enabled extensions: Add Username to Top Panel, Bluetooth battery indicator, Bluetooth Quick Connect, Dash to Dock, Hide minimized, Removable drive menu, Status area horizontal spacing, system-monitor-next, Trash, Ubuntu Appindicators, Ubuntu Dock

**Comment by G-dH** [comment]
Thanks for feedback, please install the latest version from the GitHub repository and let me know if it helped.

**Comment by zazy** [comment]
Ubuntu Dock is not in use (disabled). My fault in bug report, sorry.
Installed latest version from github and behaviour is the same. I attach a video recorded with my phone so that first launch behavior can be seen. Look at the docking bar ghosting when hiding that happens only on first launch.
Sorry for video low quality, I shared it through whatsapp to speed up not having cables with me to connect the phone.


https://github.com/G-dH/vertical-workspaces/assets/10668286/e2e1032e-1001-44c6-95f9-0d28c0d4f725

**Comment by G-dH** [comment]
I'm developing V-Shell on Ubuntu 22.04 and I'm testing it on other systems, all without this issue. Are you sure that it's not related to the Dock?

**Comment by zazy** [comment]
You are right! I used to disable all extensions but DashToDock because I found "Fix for Dash to Dock" option and taken for granted that would work. Sorry.
Disabling both extensions (Dash to dock and Ubuntu dock) makes V-Shell works as expected. The problem returns if I enable one of Dash to dock or Ubuntu dock.
The strange think is that this happens only on the first use.

**Comment by G-dH** [comment]
Without the "fix" option V-Shell would be unusable with DtD as it's overriding vital V-Shell parts. However, on my system it's ok even with DtD enabled.
Do you really need DtD? V-Shell extends capabilities of the default dash including bottom hot edge activation of the overview, which can be set to the _Satic Workspace_ _Overview Mode_ so you can use the overview instead of Dock without scaling windows..

**Comment by G-dH** [comment]
You can also try my AATWS in dock mode, it can be used from any monitor and offers better window navigation than DtD. DtD is just a source of issues and a wasted time to me ;)

**Comment by zazy** [comment]
Thank you for AATWS will give it a try!

I used to install Ubuntu 22.04 on a notebook these days and found that the problem is not caused by an installed extension. 
On the notebook I've installed the exactly same extensions I have on my workstation (where the bug is present) and on the notebook V-Shell works as expected even on the first use.

The workstation has kernel 6.1.0, intel i5-8400 CPU and and nVidia GeForce GTX 1050 Ti 
The notebook has kernel 5.19.0, AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx

I think that CPU should be excluded as the cause of the problem so maybe that V-Shell or any other extension got problems with kernel 6.1.0 and/or GeForce/nVidia drivers
